x/incentives/keeper/suite_test.go

```diff
@@ -1,8 +1,8 @@
 package keeper_test
 
 import (
+	"crypto/rand"
```

Comment:
why change of package?

Reply:
great question -- math/rand has been deprecated in favor of crypto/rand :)